Home_Challenge1_Pyton.py

```diff
@@ -0,0 +1,28 @@
+file = open("C:/Users/tomer/PycharmProjects/HomeChallenge3/Readme.txt")
+words = open("C:/Users/tomer/PycharmProjects/HomeChallenge3/Readme.txt").read().split()
+
+print(file.read())
+print(words)
+
+
+def keywithmaxval(d):
+    """ a) create a list of the dict's keys and values;
```

Comment:
docstring should contain:
1. general description of the function
2. arguments - brief description and their type
3. returns -  brief description and their type